web/template/home.gohtml

```diff
@@ -290,8 +292,266 @@
+                                                    <div>
+                                                        <template x-if="vod.HasName()">
+                                                            <a class="title" :href="course.WatchURL(vod.ID)"
+                                                               x-text="vod.Name"></a>
+                                                        </template>
+                                                        <span class="date text-xs"
+                                                              x-text="vod.FriendlyDateStart()"></span>
+                                                    </div>
+                                                    <button type="button" @click="vod.Dropdown.toggle()"
+                                                            class="md:group-hover:block md:hidden">
+                                                        <i class="fa-solid fa-ellipsis-vertical"></i>
+                                                    </button>
+                                                    <template x-if="vod.Dropdown.value">
+                                                        <div class="absolute tum-live-menu py-2 absolute top-full right-0 h-fit overflow-hidden z-50 w-56">
+                                                            {{if $user}}
+                                                                <button type="button"
+                                                                        @click="vod.Progress.ToggleWatched()"
+                                                                        class="tum-live-menu-item">
+                                                                    <i class="fa-solid mr-4"
+                                                                       :class="vod.Progress.watched
+                                                                ? 'fa-eye-slash'
+                                                                : 'fa-eye'"></i>
+                                                                    <span class="block"
+                                                                          x-text="vod.Progress.watched
+                                                                ? 'Unmark as watched'
+                                                                : 'Mark as watched'"></span>
+                                                                </button>
+                                                            {{end}}
+                                                            <button type="button" @click="copyHLS(vod)"
+                                                                    class="tum-live-menu-item">
+                                                                <i class="fa-solid fa-link mr-4"></i>
+                                                                <span class="block">Copy HLS URL</span>
```

Comment:
Copying the HLS works, but some kind of feedback that it was copied would be nice.
In the simplest case, this would just incorporate closing the drop-down.

Reply:
Good idea. Now the dropdown closes after a click on "Copy HLS".